data/README.md

```diff
@@ -1,40 +0,0 @@
-# Final Group Project
-Module 20: Final Project.  This is our group project for Data Science Bootcamp.  Below is one of the different proposed final projects.
```

Comment:
It appears your PR will delete everything we have in our README.md. Can you back out this change?

Reply:
Hi Chris, this README.md file is not the main one, this is one I created and was in the \data\ folder noting the original weather/stock idea I proposed, let me know if you still see a concern and I'll add it back.

Reply:
gotcha